test/dids/OracleResolveDriver.test.ts

```diff
@@ -124,4 +221,18 @@ test('Test 14: Valid Params passed and REST call not made, failure ', ()=>{
 
 test('Test 15: Valid Params passed and REST call made, failure UNIMPLEMENTED', ()=>{
     expect(false);//TO BE IMPLEMENTED
+});
+
+
+//didContextPush function
+test('Test 16: Invalid Option with Valid Diddoc passed, error thrown', ()=>{
+    const driver = new OracleResolveDriver();
+    expect(()=>{driver.didContextPush(new DidDocument({id: "1"}), 5)}).toThrow();
+});
+
+test('Test 17: Valid (Test) Option with Valid Diddoc passed, boolean returned', ()=>{
+    const driver = new OracleResolveDriver();
+    const result = driver.didContextPush(didDocumentInstance, 2);
+    expect(()=>{driver.didContextPush(didDocumentInstance, 2)}).not.toThrow();
+    expect(result == true);
```

Comment:
@adrianmross Should I add another test that tests a failed context push?

Reply:
Yes. One that is not valid.